iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/ratis/RatisConsensusTest.java

```diff
@@ -220,11 +220,15 @@ private void doConsensus(IConsensus consensus, ConsensusGroupId gid, int count,
           () -> {
             ByteBufferConsensusRequest incrReq = TestUtils.TestRequest.incrRequest();
 
-            ConsensusWriteResponse response = consensus.write(gid, incrReq);
-            if (response.getException() != null) {
-              response.getException().printStackTrace(System.out);
+            TSStatus response;
+            try {
+              response = consensus.write(gid, incrReq);
+            } catch (ConsensusException e) {
```

Comment:
Shall we fail the test immediately when an exception is thrown?

Reply:
It will fail in line 259